src/styles/blocks/header.scss

```diff
@@ -0,0 +1,66 @@
+.header {
+  height: $header-height;
+
+  display: flex;
+  align-items: stretch;
+  justify-content: space-between;
+  padding-right: 50px;
+}
+
+.logo {
+  display: inline-block;
+  align-self: center;
+  width: 40px;
+  height: 40px;
+  margin-left: 50px;
+}
```

Comment:
If you make it a separate block, please remove the margin and take it to the separate file
Or you can refactor it to the element (say `header__logo`)